pkg/asset/ignition/bootstrap/common.go

```diff
@@ -257,18 +258,16 @@ func (a *Common) getTemplateData(dependencies asset.Parents, bootstrapInPlace bo
 
 	// Generate platform-specific bootstrap data
 	var platformData platformTemplateData
+	var machineCidr string
 
 	switch installConfig.Config.Platform.Name() {
 	case baremetaltypes.Name:
 		platformData.BareMetal = baremetal.GetTemplateData(installConfig.Config.Platform.BareMetal, installConfig.Config.MachineNetwork, ironicCreds.Username, ironicCreds.Password)
+		machineCidr = installConfig.Config.MachineNetwork[0].CIDR.String()
```

Comment:
There's no guarantee that the bootstrap will have an IP on the first MachineNetwork, is there?